client/src/index.jsx

```diff
@@ -12,9 +12,11 @@ import getMuiTheme from 'material-ui/styles/getMuiTheme';
 // ---------- Componenets ---------- //
 import LoggedOutHome from './components/LoggedOutHome.jsx';
 import Home from './components/Home.jsx';
-import NavBar from './components/Navbar.jsx';
 import Login from './components/Login.jsx';
 import SignUp from './components/SignUp.jsx';
+import Profile from './components/Profile.jsx';
```

Comment:
Yeah it looks like the 'Home' component in the Profile.jsx file should be 'Profile' since it is being referenced that way here.